Test/wast/tee.wast

```diff
@@ -0,0 +1,29 @@
+;; poor man's tee
+;; Outputs to stderr and stdout whatever comes in stdin
+
+(module
+  (import $__fwrite "env" "_fwrite" (param i32 i32 i32 i32) (result i32))
+  (import $__fread "env" "_fread" (param i32 i32 i32 i32) (result i32))
+  (import $_get__stdin "env" "get__stdin" (param) (result i32))
+  (import $_get__stdout "env" "get__stdout" (param) (result i32))
+  (import $_get__stderr "env" "get__stderr" (param) (result i32))
+  (export "main" $main)
```

Comment:
Is this export right? I would expect it to be an error, since the actual function definition exports "main", and does not bind $main. That might be a validation bug...